Sources/SwiftttCamera.swift

```diff
@@ -142,39 +142,67 @@ extension SwiftttCamera {
     private func handleDeviceAuthorization(_ authorized: Bool) {
         if authorized {
             session = AVCaptureSession()
-            session.setSessionPresetIfPossible(.photo)
-            let device: AVCaptureDevice? = AVCaptureDevice.captureDevice(from: cameraDevice) ?? AVCaptureDevice.default(for: .video) // It's nil only in Simulator
+            session.beginConfiguration()  // Begin configuration to setup session
+            session.sessionPreset = .photo  // Set the session preset
+
+            // Use a discovery session to find and prioritize the triple camera
+            let discoverySession = AVCaptureDevice.DiscoverySession(
+                deviceTypes: [
+                    .builtInTripleCamera,
+                    .builtInDualWideCamera,
+                    .builtInDualCamera,
+                    .builtInUltraWideCamera,
+                    .builtInWideAngleCamera,
+                    .builtInTelephotoCamera
+                ],
+                mediaType: .video,
+                position: .back
+            )
+
+            guard let device = discoverySession.devices.first else {
```

Comment:
This code doesn't make use of existing utilities that exist in this project; could you take a look at `Sources/Extensions/AVCaptureDevice+Utilities.swift` and adapt & use one of those existing utilities? Looks like the only thing that needed to change was sorting the list here in a different way, maybe?:

https://github.com/rogerluan/SwiftttCamera/blob/8c418c68a20b990c94644add4906d74d94592a80/Sources/Extensions/AVCaptureDevice%2BUtilities.swift#L14-L22

Or tweaking how the session discovers the device, not sure, it's been a long time since I wrote all this 😅

Reply:
@rogerluan Sorry, I put this on the shelf a few months ago, there was something with how it was handled before that wasn't discovering the lens I had on my 15 Pro Max, and that was preventing a continuous focusing issue I had. I'll look into it again.